04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+The following example can briefly explain the dilemma:\\
+
+The closest an island from the eye of the storm, the larger the severity on that island. Therefore, the more resources are depleted. Let’s say that the disaster epicentre hits around island 1 as shown in Figure below. and that the desired common pool threshold has not been met. Island 1 would be experiencing severe damages , and island 2 and 6 some damages and the other islands low to no damage. Therefore, Island 1,2 and 6 have the risk of not surviving if they are lacking resources, whereas island 3, 4, and 5 would not be affected as much by the disaster.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Disaster eye of the storm severity.PNG}
+    \caption{Disaster Epicenter effects}
+    \label{Images:Disaster eye of the storm severity}
+\end{figure}
+
+\newpage
+\subsection{Functions within the Disaster}
+It is worth mentioning that the island's geography in infrastructure is the combination of the island number and the island’s fixed location in the archipelago.\\
+
+\begin{table}[h]
+\begin{center}
+\begin{tabular}{|p{1.1in}|p{1.1in}|p{1.1in}|p{1.1in}|} \hline
+\textbf{Functions} & \textbf{Description} & \textbf{Inputs} & \textbf{Outputs} \\ \hline
+\textit{Bernouilli Distribution \texttt{(pdfGlobal)}} & Whether a disaster is going to occur & -The probability of a disaster occurring,\newline -The Global probability variable & Determines the likelihood of a disaster occurring \\ \hline
+\texttt{PdfMag} & Exponential distribution & -The spatial PDF & Determine the disaster magnitude \\ \hline
+\texttt{Epix,EpiY} & Spatial probability distribution function & -The spatial PDF\newline -Disaster Magnitude & Determines the location of the disaster epicentre \\ \hline
+\texttt{DisasterEffect} & Island's Proximity distance to the eye of the storm & -Island geography\newline -Eye of the storm location & Determine the islands damage \\ \hline
+\end{tabular}
+\caption{\label{tab:table-name}Disaster's main functions}
+\end{center}
+\end{table}
+
+The disaster has been computed as part of the infrastructure: Each island has a fixed position, characterised using Cartesian coordinates. They are located in the archipelago which captures the collection of island geographies including the bounding regions of the entire archipelago.\\
+
+Furthermore, to determine when a disaster is going to occur, we have used the Bernoulli random variable (probability of a disaster occurring is low).\\
```

Comment:
revise -- what you mean by :
`(probability of a disaster occurring is low).` 
it is not understandable

Reply:
we have used - personal pronouns again - they have no place in academic writing (especially in the UK)

Reply:
same with unnecessary \\ at the end - does not belong at the end of every paragraph